### ISSUE TITLE
composer cannot find private repository in one project, finds it in two other projects on the same machine

### ISSUE THREAD

**Issue by adrx** [composer/composer#4760]
I have a small private github repo containing code used in three projects that I am developing on a computer running Windows. On that machine each project has a separate copy of composer. In each project the composer,json includes the repo in the "requires" section:

```
"vendorname/common": "dev-master"
```

and in a repositories section:

```
"repositories": [
      {
        "type": "vcs",
        "url": "git@github.com:username/VendornameCommon.git"
      }
    ]
```

On two of the projects composer has no problem finding and updating the dependency. On one, however, it reports that "The requested package vendorname/common could not be found in any version."

This outcome (i.e., works on two projects, not on the third) occurs even if I change the version of composer, or by clearing the composer cache. I have done as described in the troubleshooting document, but the problem remains. If I delete the entire vendor folder in the problem project and then run "php composer.phar install," the dependencies installed include the version of "vendorname/common" that is recorded in the composer.lock file. It is not the most recent version and, again, when I then run "php composer.phar update" I am told again that the package cannot be found.

On this project only, composer also fails to find a public repo on my github account that is specified in the repositories section when I run an update.

I cannot find or imagine what would be causing this. Can you suggest how I might track down what has messed up composer's ability to function in the one project?

**Comment by alcohol** [comment]
Please put the full `composer.json` of all 3 projects in a gist (unmodified).

**Comment by adrx** [comment]
Thank you for responding, and focusing me on the composer.json files again. 

I had thought the "repositories" sections in each of the files were the same. They were the same inside the section, but when I looked again at the  "repositories" section of the problem project's composer.json file I noticed that I had inserted it ust above the closing bracket of the section above it, rather than below that bracket. When I moved it to where it should have been, that solved the problem.
